src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -806,3 +809,20 @@ def __set_errors_json(self, error_count_by_operation, errors_by_operation):
         }
     # endregion
 
+    def __removed_older_complete_status_files(self, status_folder):
+        """ Retain 10 latest status complete file and remove other .complete.status files """
+        files_to_removed = []
```

Comment:
nit: rename to files_to_remove

Reply:
Saw how this is used later. Based on the usage, rename this to 'files_removed' since it is populated after the files have been removed (with os.remove) and is use to log all the removed files

Reply:
done